src/test/java/org/terasology/launcher/util/TestFileUtils.java

```diff
@@ -235,4 +271,26 @@ public void testExtract(@TempDir Path zipDir, @TempDir Path outputDir) throws IO
         assertEquals(file1Contents, Files.readAllLines(extractedFileInRoot).get(0));
         assertEquals(file2Contents, Files.readAllLines(extractedFileInFolder).get(0));
     }
+
+    private void removeAclS(AclFileAttributeView view, boolean removeRead) throws IOException {
```

Comment:
I was kinda surprised myself. I jumped to this will be easy, just get rid of the POSIX setup. Turned out that the `testCannotCreateDirectory()` is actually failing on `assert tempFolderFile.setWritable(false);`, as it returns false on Win. I'm not exactly sure why as it is using`DefaultFileSystem.getFileSystem();` and one would expect this to be implemented in that FS classes.
But I've moved my dev activities to Linux around a year ago due to these weird things going on with Win resulting in poor compatibility its lack thereof of some great tools so I'm not that much surprised.
Making it as a function was a must-have since I need it to do twice in both tests...